util/README.md

```diff
@@ -0,0 +1,21 @@
+#Rule Scraper
```

Comment:
small nit: please add spaces after the '#' I think its clearer and also its good to be consistent with what is already being followed in .gitignore and the root README